### PR TITLE
Update pytest-dash

### PR DESCRIPTION
**Pull request by T4rk1n**
Replace pytest-dash fixtures that were renamed in pytest-dash 1.0.0

The tests are still running threaded as I couldn't make the subprocess test work on circleci.

@valentijnnieman Please review.

### CODE REVIEW

**Comment by valentijnnieman** [comment]
I don't know `pytest` well enough at this point to do a proper review - discussed with @T4rk1n and since he is making changes to it anyway, he's asking someone else for a review.

**Comment by T4rk1n** [comment]
@alexcjohnson Please review

File: {{cookiecutter.project_shortname}}/tests/test_usage.py [hunk 1]
```diff
@@ -21,11 +26,6 @@ def test_render_component(dash_app, selenium):
...
-    # Wait for the output callback to complete.
-    time.sleep(1)
-
-    # Get the output component.
-    output = selenium.find_element_by_id('output')
-
-    # assert the text has changed
-    assert output.text == 'You have entered Hello dash'
+    # Wait for the text to equal, if after 10 second (default) the is not
+    # equal it will fail the test.
```
Comment (alexcjohnson) [inline {{cookiecutter.project_shortname}}/tests/test_usage.py]: ... the *text* is not equal ...
Might as well mention the `timeout` parameter by name if you're already going to the trouble of saying that 10 seconds is the default. `timeout` is pretty standard in python, but still...

**Comment by alexcjohnson** [comment]
Looks great, just one small comment-comment then 💃

**Pull request merged**